### PR TITLE
EPMRPP-35548 gy to p

### PR DESCRIPTION
**Pull request by gyurcigyurma**
@ypetya  Please look at it.  tnx

### CODE REVIEW

File: app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx [hunk 1]
```diff
@@ -21,15 +21,11 @@
...
-  AttachmentCodeModal,
-  AttachmentHarFileModal,
-  AttachmentImageModal,
-} from '.';
+import { AttachmentCodeModal, AttachmentHarFileModal, AttachmentImageModal } from '.';
 import README from './README.md';
 import harData from './example.har';
 
-storiesOf('Pages/Inside/LogsPage/AttachmentModal', module)
+storiesOf('Pages/Inside/LogsPage/AttachmentCodeModal', module)
```
Comment (ypetya) [inline app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx]: This should remain `AttachmentModal` as codeModal and harModal are part of it. Beside that it seems all good! Thanks

**Review by ypetya — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx (hunk 1 above)
Comment (gyurcigyurma) [reply app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx]: Yes, linter said that Pages/Inside/LogsPage/AttachmentModal is not exists. I thought that you removed/renamed/refactored them. 
So now only attachmentCodeModal.jsx	, attachmentHarFileModal.jsx attachmentImageModal.jsx is available.  

You export it here but no such file https://github.com/ypetya/service-ui/blob/03c3817647f591f2b426e389c6fd41c015a99e4a/app/src/pages/inside/logsPage/modals/attachmentModal/index.js#L1 
Or I missed sg? :)

Comment (ypetya) [reply app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx]: oops. Yes indeed, no such file. It is intentional! We don't need an empty modal. But the group name can stay attachmentModal in the stories group.

Comment (gyurcigyurma) [reply app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx]: Ok I'll made the changes.

Comment (gyurcigyurma) [reply app/src/pages/inside/logsPage/modals/attachmentModal/attachmentModal.stories.jsx]: @ypetya  please review

**Review by ypetya — APPROVED** [review]
Thanks!

**Pull request merged**